src/main/resources/assets/computercraft/lua/rom/startup

```diff
@@ -209,22 +209,45 @@ if fs.exists( "/rom/autorun" ) and fs.isDir( "/rom/autorun" ) then
 	end
 end
 
+local function findStartups( sBaseDir )
+    local tStartups = nil
+    local sBasePath = "/" .. fs.combine( sBaseDir, "startup" )
+    local sStartupNode = shell.resolveProgram( sBasePath )
+    if sStartupNode then
+        tStartups = { sStartupNode }
+    end
+    -- It's possible that there is a startup directory and a startup.lua file, so this has to be
+    -- executed even if a file has already been found.
```

Comment:
Yeah, I generally only use `--[[`/`]]` if I have more than 4 lines of comment, which is rarely the case.